### PR TITLE
1234 code splitting bundles

### PR DESCRIPTION
**Pull request by norisalsaadie**
[run-saucelabs]

Follow on to https://github.com/libero/reviewer/issues/1234

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -150,6 +150,7 @@
         "react-hook-form": "^4.8.2",
         "react-i18next": "^10.12.2",
         "react-router-dom": "^5.0.1",
+        "react-scripts": "^3.4.1",
```
Comment (Cooryd) [inline package.json:153]: Should this not still be a devDependency?

Comment (norisalsaadie) [reply package.json:153]: You are right. It may have returned as a result of the git cherry-pick from an older branch

Comment (norisalsaadie) [reply package.json:153]: Actually, just reviewed a new bundle created by CRA and it's actually a dependency and not a dev one. Probably best to stay inline with how react-scripts should be used.

**Comment by norisalsaadie** [comment]
Closed in favour of https://github.com/libero/reviewer-client/pull/357

**Pull request closed without merging**